resources/views/community/components/claims/finished-claims.blade.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use LegacyApp\Community\Enums\ClaimFilters;
+use LegacyApp\Community\Enums\ClaimSorting;
+
+$claimData = getFilteredClaims(
+    null,
+    ClaimFilters::AllCompletedPrimaryClaims,
+    ClaimSorting::FinishedDateDescending,
+    false,
+    null,
+    0,
+    $count
```

Comment:
recommend using named parameters as was done for new-claims.blade.php

Reply:
Adjusted in https://github.com/RetroAchievements/RAWeb/commit/c2825edbcf8c4be73c349a67619991b666c00390.

As an aside, what is your opinion on best practice for this? Do you generally reach for named parameters when the count of args exceeds a certain number?

Reply:
I generally used named parameters whenever a default parameter can be skipped. In this case, all the parameters have defaults and most can be skipped.

I sometimes also use it when it's unclear what a parameter is doing. In this case, it's not obvious what the two nulls are for without looking at the parameter names. I wouldn't use that alone as an argument to use named parameters though.

Reply:
Understood! That is helpful. With this in mind, I modified the function call to exclude the defaults ([ref](https://github.com/RetroAchievements/RAWeb/pull/1425/files#diff-6f464c0feeee2496593b4431164aa1fac3563f32f57ebb5e505cf60d9fc700f5R14)).